infrastructure/templates/datafactory/components/template.json

```diff
@@ -1871,6 +1871,26 @@
                                             "type": "String"
                                         }
                                     },
+                                    {
```

Comment:
This PR missed a couple of changes for Data Factory to work with the new GeographicLevels. So for anyone looking at this in the future, looking to see what they need to change to add / update a GeographicLevel, see the additional PR: https://github.com/dfe-analytical-services/explore-education-statistics/pull/2694